config/src/lib.rs

```diff
@@ -300,3 +324,167 @@ impl<PublicKey: VerifyingKey> Committee<PublicKey> {
             .collect()
     }
 }
+
+/// Allow us to serialize and deserialize Duration values in a more
+/// human friendly format (e.x in json files). When serialized then
+/// a string of the following format is written: [number]ms , for
+/// example "20ms". When deserialized, then a Duration is created.
+mod duration_format {
```

Comment:
We may want to export it to a different file.  In particular, I'd love to see module-level doc ( `//!`, see https://doc.rust-lang.org/reference/comments.html#examples) that makes it spectacularly clear this supports ms and s only. 


There's a chance for people to expect [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.txt) support (see also https://github.com/time-rs/time/blob/main/src/serde/rfc3339.rs), this would be out of scope, and I don't want us discovering it at deployment time.

Reply:
Good idea!  I've extracted this now to a separate file

Reply:
Regarding https://www.rfc-editor.org/rfc/rfc3339.txt you are absolutely right, definitely out of scope as I wanted something very simple for now. I've tried to make this as clear as possible on the comments.